src/GuppyDataExplorer/GuppyDataExplorer.jsx

```diff
@@ -38,10 +39,32 @@ class GuppyDataExplorer extends React.Component {
       }
     }
 
+    if (this.props.filterConfig.tabs.length > 0) {
+      const defaultFilterArr = this.props.filterConfig.tabs.reduce((accumulator, currentValue) => {
+        const df = currentValue.defaultFilters;
+        if (df && df.length > 0) {
+          return [...accumulator, ...df];
+        }
+        return accumulator;
+      }, []);
+
+      defaultFilters = defaultFilterArr.reduce((tabAccumulator, tabDefaultFilter) => {
+        if (tabDefaultFilter.field && tabDefaultFilter.values.length > 0) {
+          // if it already exists add values to it
+          if (tabAccumulator[tabDefaultFilter.field]) {
+            tabAccumulator[tabDefaultFilter.field].selectedValues.push(...tabDefaultFilter.values);
+            return tabAccumulator;
+          }
+          return { ...tabAccumulator, [tabDefaultFilter.field]: { selectedValues: tabDefaultFilter.values } };
+        }
+        return tabAccumulator;
+      }, {});
+    }
+
     this.state = {
       aggsData: {},
       filter: {},
-      initialFilterFromURL: initialFilter,
+      initialFilterFromURL: { ...defaultFilters, ...initialFilter },
```

Comment:
i don't know if we even use `explorerStoreFilterInURL` anywhere - but it's raising a couple of questions
- if I send a URL with filters, should it include the default filters?
- if I open a URL with filters, should the default filters be applied on top of those?

Reply:
if `explorerStoreFilterInURL` isn't used anywhere, maybe we can leave this as-is and document in the config doc that `explorerStoreFilterInURL` and `defaultFilters` don't work well together

Reply:
we can fix it if we ever need to

Reply:
I don't think `explorerStoreFilterInURL` has been enabled anywhere in any existing project. But in terms of the relationships of `explorerStoreFilterInURL` and `defaultFilters`, I would propose:
1. if `explorerStoreFilterInURL` is enabled, `defaultFilters` will always be placed in the URL as part of the initial filters
2. if a page is opened by pasting an URL, then portal should add those `defaultFilters` when parsing the input if the input filters from URL doesn't have them, even if this will alter the URL

Reply:
I agree with #1 but not #2, i think we should allow users to send a URL for a cohort where default filters have been changed. Otherwise the cohort when you open the URL will be different from the cohort the user meant to share.

Reply:
Hmm I see. Yes that makes sense. I agree

Reply:
so its good?, or should i change the code to make it more explicit of whats going on

Reply:
i think we should update it to do this:
- if explorerStoreFilterInURL disabled: just check the defaultFilters
- if explorerStoreFilterInURL enabled:
  - add defaultFilters to the URL by default
  - when a user lands on the explorer page from a URL with filters, respect the URL filters only and do not check the defaultFilters

It's up to you @ocshawn and @mfshao if this can be done in this PR or if it's too much scope creep, in that case let's create a separate ticket

Reply:
I think that's the right approach